db/snapshot/backup_task.cc

```diff
@@ -27,13 +27,15 @@ backup_task_impl::backup_task_impl(tasks::task_manager::module_ptr module,
                                    sstring bucket,
                                    sstring prefix,
                                    sstring ks,
+                                   sstring table,
                                    sstring snapshot_name) noexcept
     : tasks::task_manager::task::impl(module, tasks::task_id::create_random_id(), 0, "node", ks, "", "", tasks::task_id::create_null_id())
     , _snap_ctl(ctl)
     , _client(std::move(client))
     , _bucket(std::move(bucket))
     , _prefix(std::move(prefix))
     , _ks(std::move(ks))
+    , _table(std::move(table))
```

Comment:
But where is this field used in the .run() method? It will scans the whole keyspace and picks up all the tablets in it

Reply:
> But where is this field used in the .run() method? It will scans the whole keyspace and picks up all the tablets in it

As I understand it should be reviewed in [this context](https://github.com/scylladb/scylladb/pull/20685/files)

Reply:
sorry, it was a *huge* oversight. i just forgot to change it to scan the specified table only.